app/models/hathi_trust_links.rb

```diff
@@ -51,7 +51,7 @@ def work_url
   def item_url
     return item_target(hathitrust_item_id) if publicly_available?
 
-    "https://babel.hathitrust.org/Shibboleth.sso/Login?entityID=#{stanford_shib_id}&target=#{URI.encode(item_target(hathitrust_item_id))}"
+    "https://babel.hathitrust.org/Shibboleth.sso/Login?entityID=#{stanford_shib_id}&target=#{item_target(hathitrust_item_id)}"
```

Comment:
No need to encode here because encoded below. I double-checked to make sure this output was the same as before.

Reply:
I'll give those a try tomorrow. I picked one or two real HT IDs, not knowing how varied they are.

Reply:
Yeah, they're all over the place... probably created by the contributing institution. I think it's a little less wild-west than I thought, but probably worth being extra-cautious about regardless. 

```
| aub.i10592830             |
| loc.ark:/13960/t17m0qv0b  |
| bc.ark:/13960/t5s790w6p   |
| mdl.reflections.ach00001  |
| umd.31430058079171        |
| msu.31293024597860        |
```

Reply:
@cbeer thanks for that list! I'll add some test coverage and toss up a delta soon.

Reply:
@cbeer These IDs are now encoded using `ERB::Util.url_encode`. I tested some real SW URLs pointing at HathiTrust with the sort of escaping that comes out of this method, and they are routed properly when bouncing through Shib. I changed the tests to make sure we cover this.